wot-wg-2023-draft.html

```diff
@@ -85,18 +85,14 @@ <h1 id="title">PROPOSED Web of Things Working Group Charter</h1>
       The <strong>mission</strong> of the <a href="https://www.w3.org/groups/wg/wot">Web of Things Working Group</a>
       is to counter the fragmentation of the IoT through the specification of building blocks
       that enable easy integration of IoT devices and services across IoT platforms and application domains.
-      These building blocks should complement and enhance the use of existing standards, and allow WoT to
-      be used within other ecosystems and communities.
-      This reduces the initial friction for adoption and supports further industry adoption.
-      <span class="todo">This Working Group Charter covers those aspects that the
+      These building blocks complement and enhance the use of existing standards and provide a common description 
```

Comment:
```suggestion
      These building blocks complement and enhance the use of existing standards and provide a common abstraction 
```

Thing Descriptions can use Binding Templates to "describe" other standards, but profiles prescribe a new standard which can be used as a layer of abstraction on top of other standards in order to link them together.

Reply:
The word "description" was replacing "model" which was deemed to general (e.g. could also include things like specific data models for particular device types, which we are NOT doing) and also confusing.  We suggested changing it to "metamodel" (for TDs) but that was deemed confusing also; the average reviewer would not understand what we meant.  We also considered, as you suggest, "abstraction" but that was deemed as having the same problem as "model" and "meta model", e.g. being too general/confusing.  After much discussion we settled on "description" as a plain-language replacement for "metamodel".  Because we already discussed and rejected "abstraction" I won't merge this suggestion.

However... your real point is that we don't mention prescriptive deliverables and goals such as for discovery and profiles, which is a valid point, so we'll discuss whether we can add separate wording for that.